custom-words.txt

```diff
@@ -1439,7 +1439,6 @@ racyscore
 RAGRS
 RAGZRS
 randint
-randomnumbers
```

Comment:
will `rng` or `RNG` need to be added to custom-words?

Reply:
I don't think it needs, according to the successful SpellCheck.